.github/workflows/pull-request-ui.yml

```diff
@@ -3,18 +3,66 @@ name: Pull Request UI Build
 on:
   workflow_dispatch:
   pull_request:
-    paths:
-      - ".github/pull-request-ui.yml"
-      - "src/electionguard-ui/**"
     branches: [main]
 
 env:
   TARGET: Release
 
 jobs:
+  get_version_code:
+    name: Get Version Code
+    runs-on: ubuntu-22.04
+    outputs:
+      version: ${{ steps.get_version_code.outputs.version }}
+      build_number: ${{ steps.get_version_code.outputs.build_number }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Get Version Code
+        id: get_version_code
+        run: |
+          git fetch --depth=100 origin +refs/tags/*:refs/tags/*
```

Comment:
checkout only does there last commit by default so we need to fetch the tags in order to assign the version